### ISSUE TITLE
Card doesn't want to show 2022.8.2

### ISSUE THREAD

**Issue by Joshndroid** [jampez77/Multiline-Entity-Card#5]
Installation
 Home Assistant 2022.8.2 
Supervisor 2022.07.0 
Operating System 8.4 
Frontend 20220802.0 - latest 

For some reason this card wont show in lovelace as a custom card.

I have other custom cards that work, restarting a number of times wont show the card.

I have tried also searching by entity and changing the card from there, which also doesn't work

I am trying to use this with a weather entity which has a worded forecast for the day, this seems perfect for that use case.

**Comment by Joshndroid** [comment]
HACS Info (If necessary)

Integration version: | 1.26.2
Frontend version: | 20220714083628

**Comment by jampez77** [comment]
Thanks for bringing this to my attention. I plan on updating to 2022.8.2 today so I will check it out.

I'm the meantime would you be able to share your card yaml and any errors you see?

**Comment by Joshndroid** [comment]
Hey mate.
Basically it doesn't seem to show up as a card at all.
I have scrolled around to find it in the list of cards, it's not there.
I then tried copying your

type: 'custom:multiline-entity-card' and nothing worked initially. 

After about 5 restarts I have managed to manually add the type of card with an entity and it works but can't find it still in the list if searching for cards. 

Not sure if this is by design or not was all
